enom.gemspec

```diff
@@ -8,13 +8,13 @@ Gem::Specification.new do |s|
   s.email = "bensie@gmail.com"
   s.files  = %w( README.md Rakefile LICENSE ) + ["lib/enom.rb"] + Dir.glob("lib/enom/*.rb") + Dir.glob("lib/enom/commands/*.rb") + Dir.glob("test/**/*") + Dir.glob("bin/*")
   s.has_rdoc = false
-  s.add_dependency "httparty", "~> 0.10.0"
+  s.add_dependency "httparty", "~> 0.15"
   s.add_dependency "public_suffix", "~> 3.0.0"
   s.add_dependency "activesupport", "> 4.2"
   s.add_development_dependency "test-unit"
-  s.add_development_dependency "shoulda"
-  s.add_development_dependency "fakeweb"
-  s.add_development_dependency "rake", "~> 0.9"
+  s.add_development_dependency "shoulda", "~> 3.5"
+  # s.add_development_dependency "fakeweb"
```

Comment:
Can you remove this statement as it's not needed?

Reply:
This is a placeholder to add this line back after a gem is released with the necessary changes to fakeweb.  

For now, the maintainer only has them in the git repo, and you can 't use git repos in a .gemspec, so I had to put the dependency in the Gemfile.

I've put comments in both places clarifying what needs to be done when the maintainer finally pushes a gem.

On a larger note, the maintainer seems to be very intermittent with commits and releases, perhaps consider switching to another more actively maintained HTTP mocking library or a better-maintained fork of fakeweb.